### DISCUSSION TITLE
How do I see the VT control sequences?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#15901] [Q&A]
@DHowett once told me how to open a second pane in WT which would show the control sequences processed by the first pane.  I completely forgot how to do that and I doubt I could find it because it wasn't the topic of the the thread.

Would someone please tell me again how to do that.  I'll take notes this time. ;-)

**Comment by zadjii-msft** [comment]
https://github.com/microsoft/terminal/wiki/Toubleshooting-Tips#enabling-the-debug-tap

**Comment by vefatica** [comment]
Thanks!

**Comment by vefatica** [comment]
[Possibly mentioned before] Closing the tab being tapped crashes WTP.  I have crash dumps if you want them.
